Core/Helper/SystemHelper.cs

```diff
@@ -536,20 +536,14 @@ public static string GetMarketLink(string platform, string exchange, string mark
       string result = "#";
       if (platform.Equals("TradingView"))
       {
-        result = "https://www.tradingview.com/symbols/" + market.ToUpper() + "/?exchange=" + exchange.ToUpper();
-      }
-      else if (platform.Equals("TradingViewFutures"))
-      {
-        result = "https://www.tradingview.com/chart/?symbol=";
```

Comment:
Can you try using https://uk.tradingview.com/ instead of the global one, as I have been finding the global one is causing a race condition. It's the same data and functionality, so shouldn't make any difference to the user.

Reply:
sure.